docs/man/docker-cp.1.md

```diff
@@ -11,8 +12,9 @@ CONTAINER:PATH HOSTPATH
 
 # DESCRIPTION
 Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+path running the command. Paths are relative to the root of
```

Comment:
@moxiegirl I was thinking of the host running the command.  When I see "paths are relative to the root of the filesystem", I think its fair for someone to think that 'docker cp foo bar' will create /bar on my local system.  When in fact it creates it in my current dir.  But 'docker cp foo /bar' will create /bar.